zerver/openapi/python_examples.py

```diff
@@ -268,6 +268,25 @@ def create_user(client: Client) -> None:
     validate_against_openapi_schema(result, "/users", "post", "400")
 
 
+@openapi_test_function("/users/me/status:post")
+def update_status_with_clear_time(client: Client) -> None:
+    # {code_example|start}
+    # The request contains the new status and away boolean
+    request = {
+        "status_text": "on vacation",
+        "away": False,
+        "emoji_name": "car",
+        "emoji_code": "1f697",
+        "reaction_type": "unicode_emoji",
+        "scheduled_end_time": 1706625128,
```

Comment:
We can add `"scheduled_end_time": 1706625128,` to `update_status` below, instead of creating a new function. right ?

Reply:
updated!